src/Type/Comment/Connection/CommentConnectionArgs.php

```diff
@@ -189,9 +190,8 @@ private static function fields() {
 	 */
 	private static function comments_orderby_enum() {
 
-		if ( null === self::$comments_orderby_enum ) {
-
-			self::$comments_orderby_enum = new EnumType([
+		if ( null === self::$comments_orderby_enum ) :
+			self::$comments_orderby_enum = new WPEnumType([
```

Comment:
@CodeProKid  I tried to sweep and convert them all to WPEnumType. . .but I can double check to make sure I got em all